app/Http/Controllers/CampaignController.php

```diff
@@ -102,7 +103,7 @@ public function show(Request $request, Campaign $campaign)
             'campaign' => $campaign,
             'userCharacter' => $userCharacter,
             'characters' => $characters,
-            'adventures' => Adventure::filtered($search)->get(['id', 'title', 'code']),
+            'adventures' => fn () => Adventure::filtered($search)->get(['id', 'title', 'code']),
```

Comment:
from an unassuming observers perspective it would seem like it was working right, but there was actually an issue with _not_ passing a callback here. Intertia has a partial reload function that allows the front end to reload a single specified prop from the backend with new data. For that partial reloading functionality to work though, the prop needs to be passed as a callable so it can evaluate it independently